src/plugins/discover/public/application/components/sidebar/discover_sidebar.tsx

```diff
@@ -108,16 +108,20 @@ export function DiscoverSidebar({
     setFields(newFields);
   }, [selectedIndexPattern, fieldCounts, hits, services]);
 
+  const uiCapabilites = getServices().capabilities.discover;
```

Comment:
You don't need to define this variable, services is available in a memoized version on line 104
```suggestion
```